Gemfile

```diff
@@ -1,22 +1,3 @@
 source 'https://rubygems.org'
 
 gemspec
-
```

Comment:
This shouldn't be removed, see https://github.com/ruby-grape/grape-swagger-rails/blob/master/.travis.yml#L17.